### PR TITLE
 New item description UI

### PR DESCRIPTION
**Pull request by tuganimana**
I have added a new PR with the item description.

### CODE REVIEW

File: lib/ui/product/widget/product_description.dart [hunk 1]
```diff
@@ -0,0 +1,336 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_icons/flutter_icons.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:flipper/utils/HexColor.dart';
+
+class ProductDescription extends StatefulWidget {
```
Comment (richard457) [inline lib/ui/product/widget/product_description.dart:6]: We decided to not use Stateful Widget unless there is a convincing reason but I do not see any here

File: lib/ui/product/widget/product_description.dart [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+  const ProductDescription({Key key}) : super(key: key);
+
+  @override
+  _ProductDescriptionState createState() => _ProductDescriptionState();
+}
+
+class _ProductDescriptionState extends State<ProductDescription> {
+  double _checked;
+  TextEditingController quantityController = new TextEditingController();
```
Comment (richard457) [inline lib/ui/product/widget/product_description.dart:15]: These variables should be in ViewModel for using this https://github.com/FilledStacks/stacked

File: lib/ui/product/widget/product_description.dart [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+  }
+
+  void _increaseQty() {
+    setState(() {
+      quantity++;
+      titleAmount = paramsDefault + (_checked*quantity);
+    });
+  }
+
+  void _decreaseQty() {
```
Comment (richard457) [inline lib/ui/product/widget/product_description.dart:33]: FUNCTIONS SHOULD BE in view model too

File: lib/ui/product/widget/product_description.dart [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+        backgroundColor: Colors.white,
+        appBar: AppBar(
+          backgroundColor: Colors.white,
+          title: Text(
+            'Benz  FRw ${titleAmount}', // title
+            style: Theme.of(context)
+
+                .textTheme
+                .bodyText2
+                .copyWith(color: Colors.grey[800]),
```
Comment (richard457) [inline lib/ui/product/widget/product_description.dart:54]: This looks great

File: lib/ui/product/widget/product_description.dart [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+      // print(quantity);
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      child: Scaffold(
+        backgroundColor: Colors.white,
+        appBar: AppBar(
```
Comment (richard457) [inline lib/ui/product/widget/product_description.dart:46]: There is CommonAppBar , we should re-use it and modify it to match with the design if needed but let's be consistent here.

**Review by richard457 — COMMENTED** [review]
The design looks okay but the structure is not as we said, I will re-write it to match with specs and send you the change @tuganimana

**Comment by tuganimana** [comment]
Great from there I can follow the way u are using for the next work

**Comment by richard457** [comment]
flutter_icons is used but never added to pub spec!!

**Pull request merged**